### ISSUE TITLE
Wire bundle default setting

### ISSUE THREAD

**Issue by john-arvid** [john-arvid/netdesign#12]
First off you have inherited a useless default from the first version. When we ask for a fiber as a connect can it please default to 10Gbps, multimedia, LC type connector. When it is a port then it should default to 10Gbps, multimedia, SFP optics.  I have had to hand edit way too many optical blades L.

**Comment by AnotherBrian** [comment]
Just testing if I can interact with this repository. The new default value should apply to any fiber or bundle.

**Comment by john-arvid** [comment]
interaction worked, I got an email about it to. So it will be easy to follow up when comments and issues are posted.

**Comment by john-arvid** [comment]
Question:
You say that is should be the default value, then is it ok for me to assume that the default should only apply when, e.g. a port, is dropped from the stencil. But not if you copy a switch that was set to fiber and 1Gbps. So that now you have to change the port from 1Gbps to 10Gbps. ?

TLDR; Default values are only when dropping a shape from the stencil?

Respone: OK
